src/main/resources/templates/Policies/instantEmailBody.html

```diff
@@ -0,0 +1,114 @@
+{@java.time.LocalDateTime timestamp}
```

Comment:
Is(n't) that local time?

Reply:
From the docs, it's a time without the timezone. We get that from Kafka and haven't checked in details if we need to transform it.

Reply:
also, this is not a declaration of a new variable. It's just the quarkus template way of saying: "I'm going to receive a `timestamp` variable, please check that the type is `java.time.LocalDateTime`."

Reply:
A benefit from this is that templates are validated at build time that the object is being used properly.